src/com/projectkorra/projectkorra/firebending/combustion/Combustion.java

```diff
@@ -103,8 +103,8 @@ public static boolean removeAroundPoint(final Location loc, final double radius)
 	}
 
 	private void advanceLocation() {
-		ParticleEffect.FIREWORKS_SPARK.display(this.location, 5, Math.random() / 2, Math.random() / 2, Math.random() / 2, 0);
-		ParticleEffect.FLAME.display(this.location, 2, Math.random() / 2, Math.random() / 2, Math.random() / 2);
+		ParticleEffect.FIREWORKS_SPARK.display(this.location, 2, .001, .001, .001, 0);
+		ParticleEffect.CRIT.display(this.location, 3, Math.random() * 2, Math.random() * 2, Math.random() * 2, 0);

```

Comment:
Any reason for this change? imo fire particles would be fine even if they had blue fire

Reply:
Because the old combustion animation looks like a rehashed fireblast, and I decided it would be better to make it more beam like and look like the show, where we dont really see fire just "sparks"